client/home/homeReducers.js

```diff
@@ -21,12 +22,20 @@ export default (state = initialState, action) => {
       return state.setIn(['destination', 'location'], action.payload.value);
     case 'CHANGE_TRIPTYPE':
       return state.set('tripType', action.payload.tripType);
+    case 'ADD_DESTINATION':
+      const oldDests = state.get('destinations');
+      const updatedDests = oldDests.push(state.get('destination'));
+      return state.set('destinations', updatedDests);
     case 'CLEAR_DESTINATION':
       state = state.setIn(['destination', 'location'], null);
       state = state.setIn(['destination', 'startDate'], null);
       state = state.setIn(['destination', 'endDate'], null);
+      state = state.set('destinations', Immutable.List());
       state = state.set('tripType', 'Group');
```

Comment:
We can move the tripType reset to CLEAR_DESTINATIONS, and remove the reset of `destinations`. That way, CLEAR_DESTINATION will just reset the form entry values for destination location, start date, and end date. We can use CLEAR_DESTINATION after we click `Add To Trip` to clear the form but maintain the trip info.